security/webauthn/src/test/java/OpenshiftWebAuthnIT.java

```diff
@@ -0,0 +1,15 @@
+import io.quarkus.test.bootstrap.RestService;
+import io.quarkus.test.scenarios.OpenShiftScenario;
+import io.quarkus.test.services.QuarkusApplication;
+
+@OpenShiftScenario
+public class OpenshiftWebAuthnIT extends AbstractWebAuthnTest {
+
+    @QuarkusApplication
+    static RestService app = new RestService();
```

Comment:
Your Quarkus app also needs to know about database, shouldn't you set datasource url?

Reply:
OpenshiftWebAuthnIT extending from MySqlWebAuthnIT will resolve this I think